### PR TITLE
Fix meta function for aten.flip and aten.rot90

### PR DESCRIPTION
**Pull request by SherlockNoMad**
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* #88069
* #88068
* #88066
* __->__ #88065

### CODE REVIEW

File: torch/_prims/__init__.py [hunk 1]
```diff
@@ -1879,7 +1879,8 @@ def _reshape_aten(a: Tensor, shape: ShapeType) -> Tensor:
 
 def _rev_meta(a: TensorLikeType, dims: DimsSequenceType) -> TensorLikeType:
     utils.validate_dimension_indices(a.ndim, dims)
-    return TensorMeta(a)
+    out = torch.empty_like(a, memory_format=torch.preserve_format)
```
Comment (mruberry) [inline torch/_prims/__init__.py:1882]: Might as well add a comment for why this call to `empty_like` is necessary

**Review by mruberry — APPROVED** [review]
Cool!

File: torch/_prims/__init__.py (hunk 1 above)
Comment (SherlockNoMad) [reply torch/_prims/__init__.py:1882]: Will add as a tip for writing proper meta function. 
empty_like preserves meta fields, including size, stride, device, memory_layout...

**Pull request closed without merging**